### DISCUSSION TITLE
Upcoming: Release of v12.0.0

### DISCUSSION THREAD

**Discussion by georglauterbach** [docker-mailserver/docker-mailserver#3209] [Announcements]
If there are no delays due to bug reports, v12.0.0 will be released on 9 April 2023.

Related PR: #3146

If you want to support us, try the current `:edge` tag and report any incidents.

---

**v12.0.0 is our biggest release yet! Make sure to thoroughly read the changelog on release.**

**Comment by zb140** [comment]
Just a quick note from someone who had never used DMS before today.  I migrated my mailserver from a dovecot/exim4 bare-metal setup to `docker-mailserver:edge` and what I found is that it:
 - was ridiculously easy to install
 - was ridiculously easy to migrate all my old accounts and data
 - works way better than before
 - is much easier to maintain/upgrade than before

Basically what I'm saying is, I couldn't be happier! Thanks to all involved for putting in the time and effort to develop and maintain everything :smile:
